src/gpu/generic/sycl/ref_binary.hpp

```diff
@@ -94,8 +98,7 @@ struct ref_binary_t : public gpu::generic::sycl::primitive_t {
                 if (!utils::one_of(t, f32, bf16, f16, s8, u8)) return false;
             }
 
-            return IMPLICATION(utils::one_of(bf16, src0_dt, src1_dt, dst_dt),
-                    src0_dt == dst_dt && src1_dt == dst_dt);
+            return true;
```

Comment:
Why is this check removed?

Reply:
This check makes all three **src0/src1/dst** to be **bf16**. We need to allow cases like **bf16/bf16/u8**.